erlang/hello_world/README.md

```diff
@@ -6,12 +6,4 @@ The `hello_world` AtomVM application prints "Hello World" to the console and the
 
```

Comment:
I don't know if we want to address the console out situation on the Pico in the README or not, but with the default build configuration (release builds) the hello_world application will almost certainly finish successfully before the user has a chance to connect to the serial console. A workable solution is to use a paperclip or jumper wire to briefly short the `run` pin (30) to ground (28 - or any other `gnd`) after connection to reboot the device

Reply:
I'll let you or Paul handle that.

Reply:
Looks like even that workaround doesn't work, the serial is killed when it shuts down, and you still can't connect fast enough to catch the output. So hello_world technically works with a release image, but users will never be able to see the output... I think this almost renders it broken at the moment on the Pico, without a custom build.